import/index.html

```diff
@@ -0,0 +1,298 @@
+<!doctype html>
+<html class="no-js" lang="">
+
+<head>
+  <link rel="icon" type="image/svg+xml" href="./favicon.svg" />
+  <meta charset="utf-8">
+  <title>Turnkey Import</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <style>
+    body {
+    textarea {
+      width: 400px;
+      margin: 0.5em;
+      font-size: .875rem;
+      line-height: 1.25rem;
+      height: 120px;
+      color: #555b64;
+      border: none;
+      resize: none;
+      word-wrap: break-word;
+      overflow-wrap: break-word;
+      border: none;
+      font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    }
+    textarea:focus {
+      outline: none;
+      border: none;
+    }
+  }
+  </style>
+</head>
+
+<body>
+  <form>
+    <textarea name="wallet-bundle" id="wallet-bundle"></textarea>
+  </form>
+
+  <!--
+    Script defining important helpers.
+    These helpers are unit-testable, so most of the logic should be written here.
+  -->
+  <script>
+    window.TKHQ = function() {
+      /** constants for LocalStorage */
+      const TURNKEY_TARGET_EMBEDDED_KEY = "TURNKEY_TARGET_EMBEDDED_KEY"
+      /** 1 week in milliseconds */
+      const TURNKEY_TARGET_EMBEDDED_KEY_TTL_IN_MILLIS = 1000 * 60 * 60 * 24 * 7;
+
```

Comment:
Do we need an embedded key in the case of import? I think it's fine if we generate a brand new key every time: basically whichever key encrypted the payload will be in the import payload. As long as this is the case, this key can be forgotten and the iframe can completely disappear after encrypting the payload.

(and if the user refreshes the page, a new P256 key is created, that shouldn't be a problem)

EDIT: OH WAIT. I'm tired. We do need this because the import bundle will be encrypted to this embedded key right?

Reply:
Yes! This is the public key generated in the enclave and returned in the "import_bundle" returned by init_import_wallet. It's then used to encrypt the seed phrase sent in "encrypted_bundle" in import_wallet.